Frameworks/CoreGraphics/CGFont.mm

```diff
@@ -40,6 +41,9 @@
     CFRuntimeBase _base;
     ComPtr<IDWriteFontFace> _dwriteFontFace;
 
+    // Contains a value when created via Data Provider, null o/w
+    woc::unique_cf<CFDataRef> _data;
```

Comment:
That's the plan, did this to keep from having to copy the data each time this would be used.